rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapSupport.java

```diff
@@ -0,0 +1,12 @@
+package co.rsk.peg.lockingcap;
+
+import co.rsk.bitcoinj.core.Coin;
+import java.util.Optional;
+import org.ethereum.core.Transaction;
+
+public interface LockingCapSupport {
+
+    Optional<Coin> getLockingCap();
```

Comment:
Why Optional?

Reply:
It was a suggestion by @apancorb in the LockingCapStorage interface PR to avoid null values. I brought it here since it is an improvement. WDYT?